### ISSUE TITLE
Evaluation benchmarking for fine tuned models

### ISSUE THREAD

**Issue by Deeptanshu-sankhwar** [ChaosChain/chaoschain-launchpad#139]
Implement automated evaluation metrics to assess the performance of our fine-tuned LLM on EIP data. The evaluation should be run on a held-out validation set and compute metrics such as:

- ROUGE-L / ROUGE-2

- BLEU

- (Optional) BERTScore and Perplexity

Inputs should follow the format:

```
{
  "input": "<prompt>",
  "reference": "<expected completion>",
  "prediction": "<model output>"
}
```

The script should output a metrics summary in JSON or CSV, support CLI arguments for model path
